metricbeat/module/aerospike/aerospike_test.go

```diff
@@ -96,3 +98,73 @@ func TestParseInfo(t *testing.T) {
 		assert.Equal(t, test.expected, result, test.Name)
 	}
 }
+
+func getStringPointer(value string) *string {
+	return &value
+}
+
+func getBoolPointer(value bool) *bool {
+	return &value
+}
```

Comment:
Replace this with this:

https://github.com/elastic/beats/blob/8105935e24fb12a2120488290015a38f143d8471/x-pack/filebeat/input/salesforce/helper.go#L31